tests/components/zwave_mqtt/test_config_flow.py

```diff
@@ -0,0 +1,53 @@
+"""Test the Z-Wave over MQTT config flow."""
+from homeassistant import config_entries, setup
+from homeassistant.components.zwave_mqtt.config_flow import TITLE
+from homeassistant.components.zwave_mqtt.const import DOMAIN
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+
+async def test_user_create_entry(hass):
+    """Test the user step creates an entry."""
+    hass.config.components.add("mqtt")
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5460460)